res/skins/Deere (64 Samplers)/skin.xml

```diff
@@ -18,7 +18,7 @@
       <attribute persist="true" config_key="[Skin],show_8_hotcues">1</attribute>
       <attribute persist="true" config_key="[Skin],show_intro_outro_cues">1</attribute>
       <attribute persist="true" config_key="[Skin],show_coverart">1</attribute>
-      <attribute persist="true" config_key="[Skin],show_spinnies">1</attribute>
+      <attribute persist="true" config_key="[Skin],show_spinnies">0</attribute>
```

Comment:
What is the reasoning for this change? 
If you think it is an issue to show an empty Spinny in case of no OpenGL, we should always hide it in this case. 
But that is only a one time issue for Mac users, while new Windows user will have hard times to discover the spinnies.

Reply:
>  If you think it is an issue to show an empty Spinny in case of no OpenGL, we should always hide it in this case.

I disagree. Then we would need to explain somewhere in the GUI that spinnies will be disabled without OpenGL. Users will be confused if spinnies disappear when updating Mixxx without any explanation and the toggles in the skin settings do not work. With this solution, users installing Mixxx for the first time will probably not notice anything.

> new Windows user will have hard times to discover the spinnies.

I don't mind that.